apps/prairielearn/src/components/QuestionsTable.html.ts

```diff
@@ -27,24 +27,26 @@ export function QuestionsTable({
   showSharingSets,
   current_course_instance,
   course_instances,
+  qidPrefix,
   urlPrefix,
   plainUrlPrefix,
   __csrf_token,
 }: {
   questions: QuestionsPageDataAnsified[];
-  showAddQuestionButton: boolean;
-  showSharingSets: boolean;
-  current_course_instance: CourseInstance;
-  course_instances: CourseInstance[];
+  showAddQuestionButton?: boolean;
+  qidPrefix?: string;
+  showSharingSets?: boolean;
+  current_course_instance?: CourseInstance;
+  course_instances?: CourseInstance[];
   urlPrefix: string;
   plainUrlPrefix: string;
-  __csrf_token: string;
+  __csrf_token?: string;
```

Comment:
the public questions sharing page doesn't need it because there are no forms (the course questions page has a form for "add question"). So you want to have it passed in here anyway, even if it will be unused in that case?